### PR TITLE
Allow any string as key, fixes #11

### LINKED ISSUE #11: Allow syncing records whose id is not a uuid
Right now I'm using `12345678-1234-1234-1234-1234567890ab` as the key of chrome.storage.sync entry I'm trying to sync in demo/index.html. Should find a way to allow storing records with any string id into chrome.storage.sync, and still sync to a kinto server.

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: lib/chrome.js [hunk 1]
```diff
@@ -58,6 +59,16 @@ class Events {
 
 const MIN_INTERVAL = 1000;
 
+function keyToId(key) {
+  let md5Str = md5(key);
+  const parts = [];
+  [8,4,4,4,12].map(numChars => {
```
Comment (Natim) [inline lib/chrome.js:65]: Don't you want to use `node-uuid` for that?

Comment (Natim) [inline lib/chrome.js:65]: I like it like that too :)

Comment (michielbdejong) [inline lib/chrome.js:65]: afaict `node-uuid` generates random uuid's, and there's no way to use it just to add the hyphens into an md5 string, or generate a uuid deterministically starting from a given string like `key`?

Comment (Natim) [inline lib/chrome.js:65]: I remember it was possible to pass a 16 bytes array to the v4 constructor random parameter.

Comment (michielbdejong) [inline lib/chrome.js:65]: This works, but it's a bit silly to parse and then unparse: https://github.com/Kinto/kinto-chrome/commit/958e4296dbd46ae881149d37ccc287859718e5a1

Comment (michielbdejong) [inline lib/chrome.js:65]: I'll leave it with `[8,4,4,4,12].map` unless we find a nodejs module that does exactly that.

**Comment by Natim** [comment]
r+

**Comment by michielbdejong** [comment]
Thanks!

**Pull request merged**